modules/axhal/src/arch/x86_64/trap.rs

```diff
@@ -12,6 +12,9 @@ const LEGACY_SYSCALL_VECTOR: u8 = 0x80;
 const IRQ_VECTOR_START: u8 = 0x20;
 const IRQ_VECTOR_END: u8 = 0xff;
 
+const EXCEPTION_VECTOR_START: u8 = 0x0;
+const EXCEPTION_VECTOR_END: u8 = 0x1f;
```

Comment:
Rather than introducing new constants, perhaps it would be better to exclude the range of irq.

Reply:
And I think introducing a separate `is_exception` function may not be necessary? Although I believe `rustc` is smart enough to automatically `#[inline]` this function so there won't be overhead, it just makes little sense semantically.